templates/examples/qml/main.cpp.tpl

```diff
@@ -1,12 +1,16 @@
+{{- $features := .Features }}
 {{- range .System.Modules }}
 {{- $module := . }}
 {{- range $module.Interfaces }}
 {{- $interface := . }}
-#include "{{snake $module.Name}}/lib/{{ lower ( camel $interface.Name) }}.h"
-#include "{{snake $module.Name}}/lib/qml{{lower ( camel $interface.Name)}}.h"
+#include "{{snake $module.Name}}/implementation/{{ lower ( camel $interface.Name) }}.h"

```

Comment:
the lib is called "<name>_impl" but the folder is called implementation. Where for a lib both names are the same.